docs/api-docs/tax/tax-settings.md

```diff
@@ -0,0 +1,92 @@
+# Tax Settings
+
+Tax settings allow you to choose how a store handles tax calculations and price displays. For example, you can set how a store handles taxes if an enabled tax service is unavailable. You can also choose whether prices include taxes when displayed on storefront orders. For more info, see [Tax Settings](https://support.bigcommerce.com/s/article/Tax-Overview?language=en_US#tax-settings).
```

Comment:
> You can also choose whether prices include taxes when displayed on storefront orders.

This isn't quite accurate - in particular the usage of _storefront_ is bothersome. The settings available today speak to the control panel view and invoices. A shopper can access invoices from their account > order list.

If anything I'd remove the word "storefront" - this should hopefully avoid misleading people. For further context, the primary function of storefront price display is handled via _tax zones_.